include/common/spawn-primary-client.h

```diff
@@ -0,0 +1,35 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+#ifndef LABWC_SPAWN_PRIMARY_CLIENT_H
+#define LABWC_SPAWN_PRIMARY_CLIENT_H
```

Comment:
Why the own file and include rather than `common/spawn.{h,c}`? Because of the license?

Reply:
Because:

1. The license & copyright notice.
2. It's quite a specialist thing with a couple helper functions, so nice to keep them isolated/separated.
3. Makes it clear where those bits have come from for reference for us later (or for future devs) in case the cage approach develops of we want to make changes to it.